src/main/java/com/ays/assignment/model/entity/AssignmentEntity.java

```diff
@@ -68,7 +68,7 @@ public boolean isAvailable() {
     }
 
     public boolean isAssigned() {
-        return AssignmentStatus.ASSIGNED.equals(this.status);
+        return AssignmentStatus.ASSIGNED.equals(this.status) || AssignmentStatus.IN_PROGRESS.equals(this.status);
     }
```

Comment:
Kullanılan yerleri bozmuş sayılabilirsin, o yüzden buraya eklemek yerine `isInProgress()` adında bir metot yazıp servisteki kod içerisinde yeni bir filter ekleyerek oraya ekleyebilirsin

Reply:
Projede kullanılmıyordu ama okunabilirlik açısından kötü olduğu için `isInProgress()` methodu oluşturup servisteki filterı düzelttim